### DISCUSSION TITLE
Will LearnCard be able to replace my other web3 wallets?

### DISCUSSION THREAD

**Discussion by mdcoxmt** [learningeconomy/LearnCard#92] [Help] [answered]
If I have NFTs & crypto in another digital wallet, will I be able to store them instead within LearnCard? What sorts of digital currencies can I store, and what's on the roadmap?

**Comment by goblincore** [comment]
@mdcoxmt I believe @smurflo2 may be able to answer your question with more detail. From what I understand, we have a working ERC-20 implementation though to what extant or when that functionality will be added to the wallet proper is unknown to me at this moment in time.

**Comment by smurflo2** [comment, marked as answer]
@mdcoxmt Yes! Like @goblincore mentioned, we have a working ETH + ERC-20 implementation. However, it does have some limitations at the moment (we only support ETH mainnet, some ETH testnets, and using [Infura](https://infura.io/) as a provider). Supporting more networks and providers is definitely on the roadmap.

We don't have a UI for tokens in the wallet proper quite yet, but that's rapidly climbing my TODO list :smile:. This will only include the ability to transact on testnets at first and will expand in the future once we've thoroughly audited and tested it.

We don't have NFT (ERC-721) support quite yet, but it's also on our roadmap! I hope to get started on that sooner rather than later.
